docs/packaging/submitting-a-package.md

```diff
@@ -128,6 +192,6 @@ Pushing changes is not possible unless you have maintainer access. The same is a
 
```

Comment:
I can't put a comment on the line I want to because it's not technically a part of this PR, but there is a missing period after the link to the build status page.

Reply:
Fixed